.github/workflows/pr.yaml

```diff
@@ -0,0 +1,29 @@
+name: Benchmark Greeting
+on: 
+  pull_request:
+    types: [opened]
+
+# split into two jobs so it runs in parallel, even if a little redundant
+jobs:
+  make_comment:
+    if: contains(github.event.pull_request.body, 'This pull request represents a submission to the codesearchnet benchmark.')
+    runs-on: ubuntu-latest
+    steps:
+    
+    - name: Greeting
+      run: |
+        URI=https://api.github.com
+        API_VERSION=v3
+        API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
+        AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
+        MESSAGE="@${USER} Thank you for your benchmark submission!  Can you please tell us a little bit about your approach and how you trained your models etc?  What modifications, if any, did you make on the baseline models provided in this repository? Thanks!"
+        # Create a comment with APIv3 # POST /repos/:owner/:repo/issues/:issue_number/comments
+        curl -XPOST -sSL \
+        -d "{\"body\": \"$MESSAGE\"}" \
+        -H "${AUTH_HEADER}" \
+        -H "${API_HEADER}" \
+        "${URI}/repos/${GITHUB_REPOSITORY}/issues/${ISSUE_NUMBER}/comments"
```

Comment:
@hamelsmu you'll know the best, but is `issues/${ISSUE_NUMBER}` right? Should it be something with PRs?
https://developer.github.com/v3/pulls/comments/#create-a-comment
```suggestion
        "${URI}/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}/comments"
```
(don't know the actual name that `PR_NUMBER` is exposed under...)

Reply:
Good catch

Reply:
good catch ... fixing now

Reply:
Nevermind.  that endpoint is for PR Review Comments, not the pull request itself.  In GitHub, confusingly, a PR == Issue.  

https://developer.github.com/v3/pulls/:

```
The Pull Request API allows you to list, view, edit, create, and even merge pull requests. Comments on pull requests can be managed via the Issue Comments API.
```

Reply:
Same thing goes for assignees 😆 

![image](https://user-images.githubusercontent.com/1483922/67029412-13eafe00-f0c2-11e9-8d39-28d1bfc3fa65.png)